t/lib/OpenQA/SeleniumTest.pm

```diff
@@ -62,7 +62,10 @@ sub _start_gru {
 }
 
 sub enable_timeout {
+    # Element waits fail after this timeout
     $_driver->set_implicit_wait_timeout(2000);
+    # Scripts are considered stuck after this timeout
+    $_driver->set_timeout(script => 60);
```

Comment:
I would not mix these two calls to `set_implicit_wait_timeout` and `set_timeout`. I think it's a good idea to set `set_timeout` especially if the default is way higher but better do it not in `enable_timeout` which is called in our tests as well. Probably a better place is to set the timeout only where the driver instance is created, i.e. in `start_driver` further down, where we call `enable_timeout`. Also because 60 is a "magic number" better only use that as a fallback value if no environment variable is set, e.g.:

```perl
$_driver->set_timeout(script => ENV{OPENQA_SELENIUM_TEST_TIMEOUT_MS} // 60);
```

According to https://metacpan.org/pod/Selenium::Remote::Driver#set_timeout the value is taken in 60ms but I could not see the default value specified explicitly. Are you *sure* you want to set the timeout to only 60ms?

Reply:
I wanted to verify that `60ms` is fine under ideal conditions but apparently looks like this when it's hit:

    FAILED--Further testing stopped: executeScript: script timeout at /home/squamata/project/t/ui/../lib/OpenQA/SeleniumTest.pm:104

I like the idea of checking an environment variable. Will change it.